### PR TITLE
Bugfix snmpv3

### PR DESCRIPTION
**Pull request by Costya-Y**
This change is [Reviewable](https://reviewable.io/reviews/qualisystems/cloudshell-snmp/47)

### CODE REVIEW

File: cloudshell/snmp/snmp_parameters.py [hunk 1]
```diff
@@ -112,8 +112,26 @@ def __init__(
...
-        self.auth_protocol = auth_protocol
-        self.private_key_protocol = private_key_protocol
+        self.snmp_auth_protocol = auth_protocol
+        self.snmp_private_key_protocol = private_key_protocol
+
+    # For backward compatibility auth_protocol and private_key_protocol
+    # will be available through properties
+    @property
+    def auth_protocol(self):
+        return self.snmp_auth_protocol
```
Comment (saklar13) [inline cloudshell/snmp/snmp_parameters.py:122]: you can add a deprecation warning
```python
import warnings
warnings.warn("msg", DeprecationWarning, stacklevel=2)
```

Comment (Costya-Y) [reply cloudshell/snmp/snmp_parameters.py:129]: Thanks!
Added.

File: cloudshell/snmp/snmp_parameters.py [hunk 2]
```diff
@@ -112,8 +115,50 @@ def __init__(
...
-        self.auth_protocol = auth_protocol
-        self.private_key_protocol = private_key_protocol
+        self.snmp_auth_protocol = auth_protocol
+        self.snmp_private_key_protocol = private_key_protocol
+
+    # For backward compatibility auth_protocol and private_key_protocol
+    @property
+    def auth_protocol(self):
+        warnings.warn(
+            "auth_protocol is obsolete please " "use snmp_auth_protocol field instead",
```
Comment (saklar13) [inline cloudshell/snmp/snmp_parameters.py:125]:
```suggestion
            "auth_protocol is obsolete please use snmp_auth_protocol field instead",
```

File: cloudshell/snmp/snmp_parameters.py [hunk 3]
```diff
@@ -112,8 +115,50 @@ def __init__(
...
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.snmp_auth_protocol
+
+    @auth_protocol.setter
+    def auth_protocol(self, value):
+        warnings.warn(
+            "auth_protocol is obsolete please " "use snmp_auth_protocol field instead",
```
Comment (saklar13) [inline cloudshell/snmp/snmp_parameters.py:134]:
```suggestion
            "auth_protocol is obsolete please use snmp_auth_protocol field instead",
```

Comment (Costya-Y) [reply cloudshell/snmp/snmp_parameters.py]: Done

File: cloudshell/snmp/snmp_parameters.py (hunk 2 above)
Comment (Costya-Y) [reply cloudshell/snmp/snmp_parameters.py]: Done

**Pull request merged**